Resources/Textures/Clothing/Back/Backpacks/cluwne.rsi/meta.json

```diff
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Created by discord: brainfood# 7460",
```

Comment:
its better to use your github username
(same thing for all the other copyrights)